pkg/clusterbuilder/providers/capa/values/cluster_values.yaml

```diff
@@ -23,3 +23,8 @@ global:
       spotInstances:
         enabled: true
         maxPrice: 0.2960
+
+  apps:
+    externalDns:
+      values:
+        triggerLoopOnEvent: true
```

Comment:
Is this related to the unification? Without context, this feels like a general improvement to reconcile faster?!

Reply:
This and the other bit you commented on are copied over from the existing values for default-apps (as they now need to be defined on the cluster app values instead)

Reply:
What Marcus said ☝️